src/bugjira/bugjira.py

```diff
@@ -0,0 +1,73 @@
+from bugjira.broker import BugzillaBroker, JiraBroker
+from bugjira.config import Config
+from bugjira.issue import Issue
+from bugjira.util import is_bugzilla_key, is_jira_key
+
+
+class Bugjira:
+    """API abstraction layer object for a bugzilla backend and a jira
+    backend."""
+
+    def __init__(
+        self, config_path="", config_dict=None, bugzilla=None, jira=None
+    ):
+        """Init method for the Bugjira class
+
+        :param config_path: Absolute path to a json config file, defaults to ""
+        :type config_path: str, optional
+        :param config_dict: A dict containing configuration info for Bugzilla
+            and Jira instances
+        :type config_dict: dict, optional
+        :param bugzilla: An already-initialized bugzilla.Bugzilla instance,
+            defaults to None
+        :type bugzilla: bugzilla.Bugzilla, optional
+        :param jira: An already-initialized jira.JIRA instance, defaults to
+            None
+        :type jira: jira.JIRA, optional
+        """
+        self.config = None
+        if config_dict:
+            self.config = Config.from_config(config_dict=config_dict)
+        elif config_path:
+            self.config = Config.from_config(config_path=config_path)
+
+        self._bugzilla_broker = BugzillaBroker(
+            config=self.config, backend=bugzilla
+        )
+        self.bugzilla = self._bugzilla_broker.backend
+
+        self._jira_broker = JiraBroker(config=self.config, backend=jira)
+        self.jira = self._jira_broker.backend
```

Comment:
I get your reasons for wanting to expose this, I am just not 100% convinced this is the best way to do it still. However, I am not going to hold up the patch on this, we can always add a new way down the road and deprecate this if we come up with a different approach.